commit 2a493d8ef5b9f7568acdf26cd7fdf9cf70d69ee7
Author: Hui Wang <hui.wang@canonical.com>
Date:   Sun Dec 9 09:16:43 2018 +0800

    ALSA: hda/realtek - Fix the mute LED regresion on Lenovo X1 Carbon
    
    commit 6ba189c5c1a4bda70dc1e4826c58b0246068bb8d upstream.
    
    Users reported a mute LED regression on Lenovo X1 Carbon, the root
    cause is we applied the fixup of ALC285_FIXUP_LENOVO_HEADPHONE_NOISE
    to this machine, then the machine can't apply the fixup of
    ALC269_FIXUP_THINKPAD_ACPI anymore. To fix it, we chain two fixup
    together.
    
    Fixes: c4cfcf6f4297 ("ALSA: hda/realtek - fix the pop noise on headphone for lenovo laptops")
    Cc: <stable@vger.kernel.org>
    Signed-off-by: Hui Wang <hui.wang@canonical.com>
    Signed-off-by: Takashi Iwai <tiwai@suse.de>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/sound/pci/hda/patch_realtek.c b/sound/pci/hda/patch_realtek.c
index 9d0393d2be1f..8a3d0694d2e5 100644
--- a/sound/pci/hda/patch_realtek.c
+++ b/sound/pci/hda/patch_realtek.c
@@ -6395,6 +6395,8 @@ static const struct hda_fixup alc269_fixups[] = {
 	[ALC285_FIXUP_LENOVO_HEADPHONE_NOISE] = {
 		.type = HDA_FIXUP_FUNC,
 		.v.func = alc285_fixup_invalidate_dacs,
+		.chained = true,
+		.chain_id = ALC269_FIXUP_THINKPAD_ACPI
 	},
 	[ALC295_FIXUP_HP_AUTO_MUTE] = {
 		.type = HDA_FIXUP_FUNC,
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1139_2a493d8ef5b9f7568acdf26cd7fdf9cf70d69ee7.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1139_2a493d8ef5b9f7568acdf26cd7fdf9cf70d69ee7.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
